glide.yaml

```diff
@@ -70,3 +70,13 @@ import:
   version: 335da9dda11408a34b64344f82e9c03779b71673
   repo: https://github.com/openshift/glog.git
   vcs: git
+- package: github.com/stretchr/testify
```

Comment:
Did you really have to introduce a testing framework? I am not saying it does not make sense, but you could have asked. All other tests use plain Go test asserts. 

Why did you choose this framework?